bench-vote/src/main.rs

```diff
@@ -0,0 +1,331 @@
+#![allow(clippy::arithmetic_side_effects)]
+
+use {
+    clap::{crate_description, crate_name, Arg, Command},
+    crossbeam_channel::unbounded,
+    solana_client::connection_cache::ConnectionCache,
+    solana_connection_cache::client_connection::ClientConnection,
+    solana_net_utils::bind_to_unspecified,
+    solana_sdk::{
+        hash::Hash, message::Message, signature::Keypair, signer::Signer, transaction::Transaction,
+    },
+    solana_streamer::{
+        packet::PacketBatchRecycler,
+        streamer::{receiver, PacketBatchReceiver, StreamerReceiveStats},
+    },
+    solana_vote_program::{vote_instruction, vote_state::Vote},
+    std::{
+        cmp::max,
+        net::{IpAddr, Ipv4Addr, SocketAddr, UdpSocket},
+        sync::{
+            atomic::{AtomicBool, AtomicUsize, Ordering},
+            Arc,
+        },
+        thread::{self, spawn, JoinHandle, Result},
+        time::{Duration, Instant, SystemTime},
+    },
+};
+
+fn sink(
+    exit: Arc<AtomicBool>,
+    rvs: Arc<AtomicUsize>,
+    r: PacketBatchReceiver,
```

Comment:
not clear what `rvs`. one letter `r` name is not good either.
Can we use more descriptive names?

Reply:
done